integrations/docker/images/chip-cert-bins/Dockerfile

```diff
@@ -0,0 +1,273 @@
+			&& scripts/build/build_examples.py --target linux-arm64-tv-app-ipv6only build \
+			&& mv out/linux-arm64-tv-app-ipv6only/chip-tv-app out/chip-tv-app \
+			&& scripts/build/build_examples.py --target linux-arm64-tv-casting-app-ipv6only build \
+			&& mv out/linux-arm64-tv-casting-app-ipv6only/chip-tv-casting-app out/chip-tv-casting-app \
+			&& scripts/build/build_examples.py --target linux-arm64-light-ipv6only build \
+			&& mv out/linux-arm64-light-ipv6only/chip-lighting-app out/chip-lighting-app \
+			&& scripts/build/build_examples.py --target linux-arm64-thermostat-ipv6only build \
+			&& mv out/linux-arm64-thermostat-ipv6only/thermostat-app out/thermostat-app \
+			&& scripts/build/build_examples.py --target linux-arm64-ota-provider-ipv6only build \
+			&& mv out/linux-arm64-ota-provider-ipv6only/chip-ota-provider-app out/chip-ota-provider-app \
+			&& scripts/build/build_examples.py --target linux-arm64-ota-requestor-ipv6only build \
+			&& mv out/linux-arm64-ota-requestor-ipv6only/chip-ota-requestor-app out/chip-ota-requestor-app \
+			&& scripts/build/build_examples.py --target linux-arm64-lock-ipv6only build \
+			&& mv out/linux-arm64-lock-ipv6only/chip-lock-app out/chip-lock-app \
+			;; \
+        *) ;; \
+    esac
+
+RUN npm --prefix third_party/zap/repo/ ci
+RUN scripts/examples/gn_build_test_example.sh app1
+
+# Stage 3: Copy relevant cert bins to a minimal image to reduce size.
+FROM ubuntu:21.10
+ENV TZ=Etc/UTC
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+RUN apt-get update -y
+RUN apt-get install -y libssl-dev libdbus-1-dev libglib2.0-dev libavahi-client-dev avahi-utils iproute2
+WORKDIR /root/
+COPY --from=chip-build-cert-bins /root/.sdk-sha-version .sdk-sha-version
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/debug/chip-tool chip-tool
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/debug/chip-shell chip-shell
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/debug/chip-cert chip-cert
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/chip-all-clusters-app chip-all-clusters-app
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/chip-lighting-app chip-lighting-app
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/chip-tv-casting-app chip-tv-casting-app
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/chip-tv-app chip-tv-app
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/chip-bridge-app chip-bridge-app
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/thermostat-app thermostat-app
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/chip-ota-provider-app chip-ota-provider-app
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/chip-ota-requestor-app chip-ota-requestor-app
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/chip-lock-app chip-lock-app
+COPY --from=chip-build-cert-bins /root/connectedhomeip/out/app1/chip-app1 chip-app1
```

Comment:
We need need one more app built: `all-clusters-minimal-app`

On TH image it is built using this command:

```sh
scripts/examples/gn_build_example.sh      examples/all-clusters-minimal-app/linux/      examples/all-clusters-minimal-app/linux/out/all-clusters-minimal-app   chip_inet_config_enable_ipv4=false
```

Reply:
OK will add.